CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 * Added support for sorting by link's field (#672).
 * Added `OrderedRealmCollectionSnapshot` class and `OrderedRealmCollection.createSnapshot()` method. `OrderedRealmCollectionSnapshot` is useful when changing `RealmResults` or `RealmList` in simple loops.
+* Added support for adding listeners on `RealmList`.
```

Comment:
`RealmList`'s iterators stay the same. They have quite different meanings compared with `RealmResults`. e.g.: `RealmList` is actually mutable for users, but `RealmResults` is not.